indico/htdocs/js/indico/modules/reporter.js

```diff
@@ -12,7 +12,13 @@
     }
 
     global.setupReportFilter = function() {
-        $('.report-filter').dropdown({selector: '.report-column .title'});
+        $('.report-filter').each(function() {
+            var $reportFilter = $(this);
```

Comment:
no need, you use it only once, you can just do `$(this).find(..)` in the next line